Add notes for 0.90.0 (#5004)
weaveworks/eksctl: docs/release_notes/0.90.0.md

# Release 0.90.0

## Features

> - Added propagate asg tags to propagate labels and taints on creation (#4962)

Reviewer: The linked issue's title is more helpful.
```suggestion
- Automatically create ASG tag from nodegroup's labels and taints (#4962)
```

Reviewer 2: I really don't like editing release notes, we should make sure our PR titles are good enough so the release process becomes smooth for us. I'll make an exception here tho since the suggestion is nice 👌🏻

# Release 0.90.0

> ## Improvements
> 
> - Set up AWS SDK v2 and migrate STS service (#4926)

Reviewer 2: ```suggestion

## Features

- Added propagate asg tags to propagate labels and taints on creation (#4962)

## Improvements

- Set up AWS SDK v2 and migrate STS service (#4926)

### Documentation

- Update docs to mention what happens when you create an addon with no policy sources (#4999)

## Acknowledgments
Weaveworks would like to sincerely thank:
 @midgleyc 
```
